var/spack/repos/builtin/packages/gdb/package.py

```diff
@@ -56,6 +56,7 @@ def install(self, spec, prefix):
         options = ['--prefix=%s' % prefix]
         if '+python' in spec:
             options.extend(['--with-python'])
+        options.append('LDFLAGS=-L{0}'.format(spec['python'].prefix.lib))
```

Comment:
It's not always `prefix.lib`, sometimes it is `prefix.lib64`. You should instead use `spec['python'].libs.ldflags`.

Reply:
Thanks